tests/integrationtest/t/executor/index_lookup_pushdown2.test

```diff
@@ -0,0 +1,23 @@
+drop table if exists t1, t2, t3, t4, t5, t6, t7, tmp1, tmp2;
+create table t1(a int, b int, c int, d int, primary key(a, b) CLUSTERED, index i(c));
```

Comment:
I think we don't support common handle currently, does tikv side support it now?